bricks/aem-logo-wall/aem-logo-wall.html

```diff
@@ -0,0 +1,97 @@
+<template>
+  <style>
+    @import url('/styles/bricks-common.css');
+
+    :host {
+      --spacing-s: 24px;
```

Comment:
Does it make sense to define variables _inside_ a component? I'd rather have all of them in a single place, outside of all components, and only _use_ them in components.
Not a blocker for merging, that's the kind of things we can discuss once we have a first working version of the homepage.

Reply:
Where do you think they should live, styles.css? Or the only css guaranteed to load before components is bricks-common.css?